lib/tapjoy/ldap/cli/group/delete.rb

```diff
@@ -0,0 +1,36 @@
+module Tapjoy
+  module LDAP
+    module CLI
+      module Group
+        # Delete LDAP group
+        class Delete
+          def delete
+            confirm unless opts[:force]
+            puts Tapjoy::LDAP::API::Group.destroy(opts[:name])
+          end
+
+          private
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Keep a blank line before and after `private`.](https://www.codacy.com/app/ali-tayarani/ldap_tools/file/2634528068/issues/source?bid=3316970&fileBranchId=3318857#l12)